src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageDependencies.cs

```diff
@@ -114,6 +114,14 @@ public string ProjectLanguage
             get; set;
         }
 
+        /// <summary>
+        /// Optional version of the compiler API (E.g. 'roslyn3.9', 'roslyn4.0')
+        /// </summary>
+        public string CompilerApiVersion
```

Comment:
Most of the `ResolvePackageDependencies` task is legacy behavior which is disabled unless you set `EmitLegacyAssetsFileItems` to true.  See https://docs.microsoft.com/dotnet/core/compatibility/msbuild#design-time-builds-only-return-top-level-package-references.

So I don't think we need to update this task.  No one should be consuming the analyzer information that it generates, and if they do it should be OK that they don't handle multi-targeted analyzers.

Reply:
Ah, that makes sense. Thanks for the information on this. I can revert this change.